src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java

```diff
@@ -905,4 +932,34 @@ private static MethodDefinition createCallableMethod(
   private static Variable createVariable(String name, TypeNode type) {
     return Variable.builder().setName(name).setType(type).build();
   }
+
+  private static String getClientClassName(String serviceName) {
+    return String.format("%sClient", serviceName);
+  }
+
+  private static List<Reference> getGenericsForCallable(
+      CallableMethodKind kind, Method method, Map<String, TypeNode> types) {
+    if (kind.equals(CallableMethodKind.LRO)) {
```

Comment:
Just FYI that `switch` would also work here.

Reply:
sure, was considering about `switch` but since there are only three conditions, probably just `if` is more clearer. But open to refactor this later if more condition is added.